lambdas/src/apis/profiles/controllers/profiles.ts

```diff
@@ -11,6 +11,45 @@ import { WearablesOwnership } from '../WearablesOwnership'
 
 const LOGGER = log4js.getLogger('profiles')
 
+function setCacheHeaders(res: Response, profiles: ProfileMetadata[]) {
+  if (profiles.length === 0) return
+
+  let maxTimestamp = profiles[0]?.timestamp
+  for (let i = 1; i < profiles.length; i++) {
+    maxTimestamp = Math.max(maxTimestamp, profiles[i].timestamp)
+  }
+
+  if (maxTimestamp) {
+    res.header('Last-Modified', new Date(maxTimestamp).toUTCString())
```

Comment:
Please add a comment on how we will be using this headers

Reply:
These headers are actually HTTP standard: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Last-Modified

I can add a link to the documentation though. That'll probably be useful